framework/src/modules/random/endpoint.ts

```diff
@@ -133,4 +135,21 @@ export class RandomEndpoint extends BaseEndpoint {
 
 		return { height: usedHashOnion.height, count: usedHashOnion.count, seed };
 	}
+
+	public async setHashOnionUsage(ctx: ModuleEndpointContext): Promise<void> {
+		validator.validate<SetHashOnionUsageRequest>(setHashOnionUsageRequest, ctx.params);
+		const { seed, count, distance, height } = ctx.params;
```

Comment:
i think `seed` and `distance` can be set from the `setHashOnion` function, so they are not necessary. Also, `count` here is used in total # of hashes and # of hashes used